core/src/store.rs

```diff
@@ -1,10 +1,10 @@
-use crate::error::BlockError;
+use crate::error::StoreError;
```

Comment:
I'm not sure if the storage part should really be part of `core`. But as it currently already is, it is outside of scope for this PR and we should discuss it on a separate issue.

Reply:
do you want to open an issue for it?

Reply:
Done: https://github.com/ipfs-rust/rust-ipld/issues/38

Reply:
I moved it out of core, is that good enough?

Reply:
Yes, sorry, that's good enough. I thought misread `src` for being `core`.